src/java.net.http/share/classes/jdk/internal/net/http/Http2ClientImpl.java

```diff
@@ -57,7 +56,7 @@ class Http2ClientImpl {
             Utils.getDebugLogger("Http2ClientImpl"::toString, Utils.DEBUG);
 
     private final HttpClientImpl client;
-    private volatile boolean stopping;
+    private boolean stopping;
```

Comment:
before this line and before line 68 - could you please add a comment that says:
```
// only accessed from within synchronized blocks
```

Otherwise it looks reasonable to me.

Reply:
Done